controlplane/kubeadm/api/v1alpha4/kubeadm_control_plane_types.go

```diff
@@ -143,6 +143,11 @@ type KubeadmControlPlaneStatus struct {
 	// +optional
 	Replicas int32 `json:"replicas,omitempty"`
 
+	// Version represents the minimum Kubernetes version for the control plane machines
+	// in the cluster
```

Comment:
Nit
```suggestion
	// in the cluster.
```

Reply:
Just fyi. I'm working on getting the godot linter in shape so we can use it for all comments (TBD if that's a good idea, but we will find out :))